shared/chat/inbox/row/big-team-channel.tsx

```diff
@@ -3,24 +3,38 @@ import * as Kb from '@/common-adapters'
 import * as React from 'react'
 import * as RowSizes from './sizes'
 import * as T from '@/constants/types'
+import * as RPCChatTypes from '@/constants/types/rpc-chat-gen'
 
 type Props = {
   layoutChannelname: string
   navKey: string
   selected: boolean
+  layoutSnippetDecoration?: RPCChatTypes.SnippetDecoration
```

Comment:
this missing plumbing helps take the internal cache and stop the flash we always see